pyproject.toml

```diff
@@ -79,16 +79,15 @@ florence_2 = [
 paligemma_2 = [
     "peft>=0.12",
     "torch>=2.4.0",
-    "transformers<4.49.0",
+    "transformers>=4.48.3",
```

Comment:
@onuralpszr why do we make it so restrictive if PaliGemma 2 was released in [`transformers==4.47.0`](https://github.com/huggingface/transformers/releases/tag/v4.47.0)?

Reply:
why not?

```
"transformers>=4.47.0",
```

Reply:
I used bug fixed version limit